### ISSUE TITLE
No issue_template 

### ISSUE THREAD

**Issue by penrowd** [excalidraw/excalidraw#8015]
### What documentation needs to change? 
There are currently no guidelines to follow when opening an issue. 

### Where is it located?
N/A

### What is wrong with it? How can it be improved?
I believe adding a template can help organize issues and ensure all important information is included. 

For example, bugs, product, security, documentation could all be headers.

**Comment by ad1992** [comment]
I think there was a PR - https://github.com/excalidraw/excalidraw/pull/4619 but unfortunately, it wasn't shipped. 
We need to get back to it and update it

**Comment by dwelle** [comment]
Yeah, originally we didn't implement the template to not add friction, but nowadays a bit of added friction might help :)
